.claude/skills/flask-soc-audit/scripts/find_imports.sh

```
@@ -0,0 +1,86 @@
+echo "--- Config → object instantiation (V-CF1) ---"
+grep -rn "= .*()$\|= .*(\s" "$ROOT/main_app/config" 2>/dev/null | grep -v "^.*#" | grep -v "class\|def\|super\|dict\|list\|tuple\|str\|int\|bool\|None\|True\|False" || echo "(none)"
+
+# Background workers importing routes (V-BG1)
+echo ""
+echo "--- Background Jobs → Routes (V-BG1) ---"
+grep -rn "from.*app_routes\|import.*app_routes" "$ROOT/main_app/background_jobs" 2>/dev/null || echo "(none)"
+
+# Workers missing app context (V-BG3) — heuristic
+echo ""
+echo "--- Background Jobs: db.session without app_context (V-BG3 heuristic) ---"
+for f in $(find "$ROOT/main_app/background_jobs" -name "*.py" 2>/dev/null); do
```

Shellcheck (0.11.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)